bundle-workflow/src/test_workflow/transform_MensorResults.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+import json
+
+import json2table
+
+
+class JsonTransformation:
+
+    def convert_to_html(self, json_file, path_to_jsonfile):
```

Comment:
This interface seems a little clunky - please add comment to clarify.

What about accepting only the data object and returning a str that is html, let callers write it to a file or not.

Reply:
As we are going to store both the html file as well as the json file in the S3 bucket, correct me if I am wrong but won't it be much simpler to store the output generated into the file directly and then working on that file